docs/src/development/variables/_index.md

```diff
@@ -0,0 +1,198 @@
+    "debug_mode": "1"
+}
+```
+
+While in the `feature-x` environment, it looks like this:
+
+```json
+{
+    "system_name": "Spiffy",
+    "system_version": "1.7",
+    "api_key": "def456",
+    "debug_mode": "1"
+}
+```
+
+To get a visual overview of which variables are overridden in an environment,
+navigate in the management console to that environment's variables settings.
+This example shows how it looks within the `feature-x` environment:
+
+<!-- vale Vale.Spelling = NO -->
+<!-- spelling turned off because of the "api_key" -->
+![The console showing the variables split into environment and project ones, with the environment variables `api_key` and `system_version` labeled as overridden and `debug_mode` as inherited the project variable `system_version` labeled as inactive.](/images/management-console/variables-overridden.png "0.5")
+<!-- vale Vale.Spelling = YES -->
+
+Project variables that conflict with environment variables are labeled as **Inactive**.
+Environment variables are labeled as **Inherited** when they get their value from a parent environment
+and as **Overridden** when there is a conflict and the parent environment's value doesn't apply.
+
+## Variable prefixes
+
+Certain variable name prefixes have special meanings.
+Some are defined by Platform.sh and apply automatically.
+Others are just available as a convention for your application code to follow.
+
+### Top-level environment variables
+
+By default, project and environment variables are only added to the `$PLATFORM_VARIABLES` environment variable.
+You can also expose a variable as its own environment variable by giving it the prefix `env:`.
+
+For example, the variable `env:foo` creates an environment variable called `FOO`.
+(Note the automatic upper-casing.)
+
+```bash
+platform variable:create --name env:foo --value bar
+```
+
+You can then access that variable directly in your app container:
+
+```bash
+$ echo $FOO
+bar
+```
+
+### PHP-specific variables
+
+Any variable with the prefix `php:` is added to the `php.ini` configuration for all PHP-based application containers in the project.
+
+For example, an environment variable named `php:display_errors` with the value `On` is equivalent to placing the following in `php.ini`:
+
+```ini
+display_errors = On
+```
+
+This feature is primarily useful to override debug configuration on development environments,
+such as enabling errors and configuring the Xdebug extension.
+
+To apply a setting to all environments or have a setting differ among multiple PHP containers in one project,
+specify the variables in the `.platform.app.yaml` file for your application.
+See the [PHP configuration page](../../languages/php/ini.md) for more information.
+
+### Framework-specific variables
+
+For specific frameworks, you can implement logic to override global configurations with environment-specific variables.
+
+The Drupal templates show examples of logic mapping variables to Drupal's configuration system.
```

Comment:
```suggestion
The [Drupal templates](https://github.com/platformsh-templates/?q=drupal) show examples of logic mapping variables to Drupal's configuration system.
```

Reply:
The specific relevant templates are linked right below so I don't think the more general link is necessary here.